deploy/manifests/prod/us-east-2/tenant/storetheindex/indexstar/deployment.yaml

```diff
@@ -21,13 +21,18 @@ spec:
             - '--providersBackends=http://oden-indexer:3000/'
             - '--providersBackends=http://kepa-indexer:3000/'
             - '--providersBackends=http://dido-indexer:3000/'
+            - '--providersBackends=http://arya-indexer:3000/'
+            - '--providersBackends=http://bala-indexer:3000/'
+            - '--providersBackends=http://cera-indexer:3000/'
             - '--backends=http://dhfind.internal.prod.cid.contact/'
             - '--backends=http://dhfind-helga.internal.prod.cid.contact/'
             - '--backends=http://dhfind-porvy.internal.prod.cid.contact/'
             - '--backends=http://oden-indexer:3000/'
+            - '--backends=http://dhfind-stateless/'
```

Comment:
Does it not need a port number?

Reply:
nope. exposed on port 80 which is default for HTTP.